README.md

```diff
@@ -12,7 +12,9 @@
 ```js
 var log = require('ak-logger');
 
-log.debug("DEBUG LOGS", debug);
+const person = { name: 'Gustavo', age: 23 }
+
+log.debug("DEBUG LOGS", person);
 
 ## Result
```

Comment:
Thank you @gustafsilva  for your contribution.

Just a small change needed here, so since you are logging person object now the result will also change.
So can you update the result also.
Also since its json object, it will log it as `[object]`
In the library there is third parameter passed to this function which is a boolean value to tell the function whether to stringify the value or not.
you might wanna consider that also.

Also after seeing this I realised. we need examples for both types of logging both for json and normal string.
I will create a new issue for this. for now you can change the result as per the data you are logging.

Reply:
Also. can you link the issue number here by adding # followed by the issue number. it will create a reference between both.

Reply:
I was looking at this now @arshadkazmi42 , thanks for linking the pull request with issue, it's my first contribution.
Will i set the example of 'How to use?' and add an example with the flag `isStringify=true`

Reply:
I think for this, you can just add a string value as log. Change the person value from json to string and update the #result part to match the input logged value

you can setup the project in your local machine and try it out.

and for passing true as third parameter to function for stringify you can create another PR as i have just now created a new issue for that.